controllers/movieController.js

```diff
@@ -0,0 +1,59 @@
+
+const movieService = require('../services/movieService');
+
+const errorhandler = (err,res) => {
+    return res.status(err.statusCode || 500).json({ message : err.message });
+};
+
+const getDetail = async (req, res) => {
+    try {
+        const { movieId } = req.params
+        
+        const listByMovieId = await movieService.getMovieDetail(movieId);
+        return res.status(200).json(listByMovieId);
+       
+    } catch(err) {
+        errorhandler(err, res);
+    }
+};
+
+const getMain = async (req, res) => {
```

Comment:
```suggestion
const getMovies = async (req, res) => {
```
말씀드렸듯이 페이지 단위가 아니라 데이터 단위 개발입니다. 실제로 개발도 데이터 단위로 하셨구요!

Reply:
네 반영하겠습니다